public/js/controllers/game.js

```diff
@@ -121,14 +121,21 @@ angular.module('mean.system')
     };
 
     $scope.startGame = function() {
-      game.startGame();
-    };
+      const players = game.players;
+      const gameID = game.gameID;
+       let friends = [];
+       players.forEach(player => {
```

Comment:
This block of code is not necessary either

Reply:
Done